build/lender/Lender.sol

```diff
@@ -364,6 +364,47 @@ contract Lender {
       return returned;
   }
 
+  /// @dev lend method signature for Notional
+  /// @param p value of a specific principal according to the Illuminate Principals Enum
+  /// @param u address of an underlying asset
+  /// @param m maturity (timestamp) of the market
+  /// @param a amount of principal tokens to lend
+  /// @return uint256 the amount of principal tokens lent out
+  function lend(uint8 p, address u, uint256 m, uint256 a) public returns (uint256) {
+      // Instantiate market and tokens
+      address[9] memory markets = IMarketPlace(marketPlace).markets(u, m); 
```

Comment:
var name nit -- its a single market , not a "markets". i'd say change that later at some point, not super important.